src/core/components/hv-navigator/index.tsx

```diff
@@ -23,6 +23,38 @@ const Stack = NavigatorService.createStackNavigator<Types.ParamTypes>();
 const BottomTab = NavigatorService.createBottomTabNavigator();
 
 export default class HvNavigator extends PureComponent<Types.Props> {
+  /**
+   * Encapsulated options for the stack screenOptions
+   */
+  stackScreenOptions = (
+    route: Types.ScreenParams,
+  ): {
+    headerMode: 'float' | 'screen' | undefined;
+    headerShown: boolean;
+    title: string | undefined;
+  } => ({
```

Comment:
could we define this as a type alias in `Types`, to make the method signature more readable?